LEGO1/lego/legoomni/include/legoanimactor.h

```diff
@@ -4,15 +4,14 @@
 #include "decomp.h"
 #include "lego/sources/anim/legoanim.h"
 #include "legopathactor.h"
-#include "mxstl/mxstl.h"
 
 // SIZE 0x20
 struct LegoAnimActorStruct {
-	LegoAnimActorStruct(float p_float, LegoAnim* p_LegoAnimPtr, LegoROI** p_roiMap, MxU32 p_numROIs);
+	LegoAnimActorStruct(float p_float, LegoAnim* p_animTreePtr, LegoROI** p_roiMap, MxU32 p_numROIs);
 	~LegoAnimActorStruct();
 	float GetDuration();
 	float m_unk0x00;         // 0x00
-	LegoAnim* m_AnimTreePtr; // 0x04
```

Comment:
If a name is confirmed through the 1996 source or beta asserts but doesn't pass `ncc`, you should add it as an exception to this file: https://github.com/isledecomp/isle/blob/master/tools/ncc/skip.yml  and it will work fine